modin/experimental/backends/omnisci/query_compiler.py

```diff
@@ -352,6 +352,8 @@ def _set_columns(self, columns):
 
     def fillna(
         self,
+        squeeze_self=False,
```

Comment:
Why are `squeeze_self` and `squeeze_value` added to the method if these are not used in any way?

Reply:
This is because definition in the base class now has these parameters.